integration/test/basic/basic_test.go

```diff
@@ -34,6 +34,15 @@ func TestChartLifecycle(t *testing.T) {
 		},
 	}
 
+	chartConfigValues := chartconfig.ChartConfigValues{
+		Channel:   "5-5-beta",
+		Name:      "tb-chart",
+		Namespace: "giantswarm",
+		Release:   "tb-release",
+		//TODO: fix this static VersionBundleVersion
```

Comment:
Should we have a TODO issue for that?

Reply:
I will do this imminently, so a TODO issue is overhead IMO.